### PR TITLE
Creating an ARM Template for Bonsai deployment

### PR DESCRIPTION
**Pull request by HamzaBoukraa**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [X] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog
* Creating an ARM Template for Bonsai deployment

### CODE REVIEW

File: 101-fast-bonsai/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    }
+  },
+  "resources": [
+    {
+      "type": "Microsoft.AutonomousSystems/workspaces",
+      "apiVersion": "2020-05-01-preview",
+      "name": "[parameters('name')]",
+      "location": "[parameters('location')]",
+      "identity": {
+        "type": "systemAssigned"
```
Comment (bmoore-msft) [inline 101-fast-bonsai/azuredeploy.json:30]: how is this used?

Comment (HamzaBoukraa) [reply 101-fast-bonsai/azuredeploy.json:30]: I added a description of Bonsai in the README. Should I add the parameters also in the README.md?

Comment (bmoore-msft) [reply 101-fast-bonsai/azuredeploy.json:30]: I'm referring the the managed identity - how is that used?  It's never assigned perms to anything...

Comment (HamzaBoukraa) [reply 101-fast-bonsai/azuredeploy.json]: Indeed, not used (and removed).

File: 101-fast-bonsai/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      "type": "string",
+      "defaultValue": "my-first-bonsai",
+      "metadata": {
+        "description": "The name of the Bonsai suite instance."
+      }
+    },
+    "location": {
+      "type": "string",
+      "allowedValues": [
+        "West US",
```
Comment (bmoore-msft) [inline 101-fast-bonsai/azuredeploy.json:15]: is this supported in USGov?  If not, we need to add the environments property to metadata.json... if so, need to update the allowedValues here

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

**Comment by bmoore-msft** [comment]
Closing abandoned request - please reopen if you can address the feedback.

**Pull request closed without merging**